Tests/Microsoft.AppCenter.Test.Functional/ExpectedData.cs

```diff
@@ -11,5 +12,6 @@ internal class ExpectedData
         internal HttpResponse Response { get; set; }
         internal Func<RequestData, bool> Where { get; set; }
         internal TaskCompletionSource<RequestData> TaskCompletionSource { get; set; }
+        internal CancellationTokenSource cancellationTokenSource { get; set; }
```

Comment:
```suggestion
        internal CancellationTokenSource СancellationTokenSource { get; set; }
```

Reply:
Fixed in separate [PR's commit](https://github.com/microsoft/appcenter-sdk-dotnet/pull/1312/commits/35970844bebf672221b1e338b41c1afa079976e8)